tests/reports.test.js

```diff
@@ -95,7 +95,24 @@ var someData = {
       {'created_at':'2015-09-05T00:00:00.000Z'},
       {'created_at':'2015-09-06T00:00:00.000Z'},
       {'created_at':'2015-09-07T00:00:00.000Z'}
+    ],
+    '/api/v1/devicestatus.json&find[created_at][$gte]=2015-08-08T00:00:00.000Z&find[created_at][$lt]=2015-09-07T23:59:59.000Z?find[openaps][$exists]=true&count=1000': [
+      {
+        "openaps": {
+            "suggested": {
+                "temp": "absolute",
+                "bg": 67,
+                "tick": "+6",
+                "eventualBG": 145,
+                "snoozeBG": 145,
+                "reason": "BG 67<74.5, delta 6>0; no high-temp to cancel",
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Mixed double and single quotes.](https://www.codacy.com/app/Nightscout/cgm-remote-monitor/file/2294742909/issues/source?bid=2452379&fileBranchId=3191689#l108)